### PR TITLE
fix: Update GroupProfile to read from properties over deprecated info aspect

### PR DESCRIPTION
**Pull request by darapuk**
## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Comment by darapuk** [comment]
This PR updates where a groups email, description and display name are read from.
The existing info field has been deprecated in favor of properties in the backend(GraphQL), but that change have not been propagated to the frontend.

Deprecation reference:
https://github.com/datahub-project/datahub/blob/fd3723ac7ef0da3e70563a81733b2de317d9b678/datahub-graphql-core/src/main/resources/entity.graphql#L2577

File: datahub-web-react/src/app/entity/group/GroupProfile.tsx [hunk 1]
```diff
@@ -89,11 +89,20 @@ export default function GroupProfile() {
     // Side bar data
     const sideBarData = {
         photoUrl: undefined,
-        avatarName: data?.corpGroup?.info?.displayName || data?.corpGroup?.name,
-        name: data?.corpGroup?.info?.displayName || data?.corpGroup?.name || undefined,
-        email: data?.corpGroup?.editableProperties?.email || undefined,
+        avatarName:
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/entity/group/GroupProfile.tsx:92]: actually, here we should use 

entityRegistry.getDisplayName(EntityType.CorpGroup, data?.corpGroup?)

File: datahub-web-react/src/app/entity/group/GroupProfile.tsx [hunk 2]
```diff
@@ -89,11 +89,20 @@ export default function GroupProfile() {
...
+            data?.corpGroup?.properties?.displayName ||
+            data?.corpGroup?.name ||
+            data?.corpGroup?.info?.displayName ||
+            undefined,
+        name:
+            data?.corpGroup?.properties?.displayName ||
+            data?.corpGroup?.name ||
+            data?.corpGroup?.info?.displayName ||
+            undefined,
+        email: data?.corpGroup?.editableProperties?.email || data?.corpGroup?.properties?.email || undefined,
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/entity/group/GroupProfile.tsx:102]: Thank you!

File: datahub-web-react/src/app/entity/group/GroupProfile.tsx [hunk 3]
```diff
@@ -89,11 +89,20 @@ export default function GroupProfile() {
...
         slack: data?.corpGroup?.editableProperties?.slack || undefined,
-        aboutText: data?.corpGroup?.editableProperties?.description || undefined,
+        aboutText:
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/entity/group/GroupProfile.tsx:104]: Thank you!

File: datahub-web-react/src/app/entity/group/GroupProfile.tsx (hunk 1 above)
Comment (jjoyce0510) [reply datahub-web-react/src/app/entity/group/GroupProfile.tsx:92]: We can do this in another PR. for now this lgtm.

**Review by jjoyce0510 — APPROVED** [review]
Huge thank you! We should have caught this during Group Profile redesign.

**Pull request merged**